### PR TITLE
🔧(labels) add missing 'release' label

### LINKED ISSUE #5: Add a release label
For PR containing only a release having a label `release` could be great. Otherwise I never which one to use.

### PR DESCRIPTION
**Pull request by jmaupetit**
## Purpose

When submitting a PR for a release, there is no real label that fits with this purpose.

## Proposal

- [x] add the `release` label with `FUN` label color

Fixes #5

### CODE REVIEW

File: github/labels/default.json [hunk 1]
```diff
@@ -47,6 +47,10 @@
     "name": "FUN",
     "color": "006b75"
   },
+  {
+    "name": "release",
+    "color": "006b75"
```
Comment (lunika) [inline github/labels/default.json]: It's the same color as `FUN` label, is it what you want ?

Comment (jmaupetit) [reply github/labels/default.json]: Yes! Unless you have another suggestion?

Comment (lunika) [reply github/labels/default.json]: Yes something green, maybe like `196811` ?

Comment (jmaupetit) [reply github/labels/default.json]: Done.

**Review by lunika — APPROVED** [review]

**Pull request merged**